.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,8 @@
+---
```

Comment:
Potentially a work item for our own team, but a separate form for feature requests would make sense as well, otherwise, lgtm